frontend/javascripts/oxalis/view/input_catcher.tsx

```diff
@@ -146,7 +146,10 @@ function InputCatcher({
   );
 
   return (
-    <div className={`inputcatcher-border ${viewportID}`}>
```

Comment:
Are these `viewportID`s used anywhere else besides the screenshot tests? In the LESS styles? Or `document.getElementById()`?

Reply:
If you mean the `screenshot_target_inputcatcher_*` ids, then, no, they are only used in the screenshot tests. Initially, I tried to avoid these additional IDs, but couldn't find a good way to get the correct DOM elements with puppeteer.